libchisel/src/binaryenopt.rs

```diff
@@ -29,6 +31,21 @@ impl<'a> ChiselModule<'a> for BinaryenOptimiser {
     }
 }
 
+impl ModuleConfig for BinaryenOptimiser {
+    fn with_defaults() -> Result<Self, ModuleError> {
+        Ok(BinaryenOptimiser::O0)
```

Comment:
Wouldn't it make more sense to do `-O2` or the highest level which doesn't sacrifice size for speed?